libs/datasets/timeseries.py

```diff
@@ -74,16 +74,16 @@ def latest_values(self, aggregation_level=None) -> pd.DataFrame:
 
         Return: DataFrame
         """
-        if not aggregation_level:
+        if aggregation_level is None:
```

Comment:
Why this change?

Reply:
No particular reason other than touching code to do with aggregation_level and thinking it is a little cleaner, but I can also see why having the `is None` on its own is better.